packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/StaffList.tsx

```diff
@@ -123,6 +277,11 @@ const HelpingFor: React.FC<HelpingForProps> = ({ studentName, helpedAt }) => {
         <span>
           Helping{' '}
           <span className="text-blue-400">{studentName ?? 'a student'}</span>{' '}
+          {extraTAStatus === ExtraTAStatus.HELPING_IN_ANOTHER_COURSE ? (
+            <span className="text-red-400">in another course </span>
+          ) : extraTAStatus === ExtraTAStatus.HELPING_IN_ANOTHER_QUEUE ? (
+            <span className="text-red-400">in another queue </span>
+          ) : null}
```

Comment:
these having the same stylization makes me think that we could do something like this:

```suggestion
          {[ExtraTAStatus.HELPING_IN_ANOTHER_COURSE. ExtraTAStatus.HELPING_IN_ANOTHER_QUEUE].includes(extraTAStatus) ? (
            <span className="text-red-400">{extraTAStatus.replace("Helping student ","")] </span>
          ) : null}
```

but that's just a thought